features/step_definitions/api/alerts_steps.rb

```diff
@@ -1,9 +1,9 @@
-# -*- coding: utf-8 -*-
+# frozen_string_literal: true
```

Comment:
With Ruby 2.6 now perhaps we don't need this anymore. But it's not an issue in any case.